rfc/unionPartitionAvailability.md

```diff
@@ -0,0 +1,50 @@
+# Enable union of partition availability
+
+This RFC implements [Github Issue 627](https://github.com/yahoo/fili/issues/627).
+
+## Goals
+
+Enable query to multiple datasources with non-intersecting data availabilities to see an union view of availabilities, 
+instead of an intersection.
+
```

Comment:
Not exactly.

Partition availability already has a concept of 'determine participating tables' and 'intersect the availability of participating tables'.  The behavior we should be thinking of is to have the underlying tables limiting the times in which they participate.

So if DS1 in included for filter set X
and DS 2 is also included for filter set X

If DS2 is defined as starting on interval 2018/FUTURE
then the availability is

Interval: THE PAST/2018
Intersect: all tables in range (DS1)
Interval 2018/The Future
Intersect: all tables in range (DS1, DS2)

Reply:
So in the case of say all of YAMP and BRX together:

2016/2017
INTERSECT: BR-US, YAMP-US
INTERSECT: 2017/2018
BR-US, YAMP-US, YAMP-EMEA, YAMP-AIPAC, YAMP-AUNZ

Reply:
@michael-mclawhorn If I understand you correctly, we are essentially filtering table columns and then intersecting, instead of unioning all availabilies, right?